### DISCUSSION TITLE
Import from various sources

### DISCUSSION THREAD

**Discussion by newtogithub-mike** [georgegebbett/recipe-buddy#448] [Ideas]
Hi I am total beginner in github, not experienced in grocy but very interested in the software.

I searched long time for a way to keep track of all my recipes from various sources. So I have books, magazines, Pdf, and websites - long search led me to "Recipe Keeper Pro" because I can use it on the phone (Android), sync it with my windows PC - and I can sit on the couch, browse one of my precious cook books, find something I like and just take a picture of it and define on screen what is ingredients, what is description and which photo I want to make it look nice. Also the recipes are scaleable, when there are more people coming around for dinner for example (very handy in my eyes).
So this app is giving a lot of options to import recipes, but I have no Idea how to get their "recipe Keeper .zip file" into grocy.

for me it would be a gamechanger being able to have the recipes there and just connect the items from grocy to the imported recipes.

thanks very much and I am looking forward to your thoughts

Michael

**Comment by jannavischer** [comment]
Other, but similar wish: 
Importing recipes from the Nextcloud Recipes App. They already should have a reasonable format and provide an API (https://nextcloud.github.io/cookbook/dev/api/), but I am not sure how the verification/login could work.
